styles.css

```diff
@@ -0,0 +1,117 @@
+body {
+  background-image: url('header-shapes mobile@2x.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+header { /* HEADER CONTAINER */

```

Comment:
- [ ] Well done using comment lines to describe the code 💯

Reply:
Thanks!